src/test/java/net/apnic/rdapd/autnum/controller/AutnumHistoryRouteControllerTest.java

```diff
@@ -54,11 +54,13 @@ public void indexLookupHasResults()
 
         mvc.perform(get("/history/autnum/1234"))
             .andExpect(status().isOk())
-            .andExpect(RDAPControllerTesting.isRDAP());
+            .andExpect(RDAPControllerTesting.isRDAP())
+            .andExpect(RDAPControllerTesting.isRDAPHistoryResult());
 
         mvc.perform(get("/history/autnum/123456789"))
```

Comment:
{suggestion} Unless I am missing something here, testing both ASNs "1234" and "123456789" makes no difference since the mock is set to return the same result independent of the the parameter, assuming it is a valid parameter of course. So having these two tests is confusing in this context. I would suggest removing this second invocation.